src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -228,3 +230,23 @@ def prepare_or_run_pipeline(
                 delay=POLLING_DELAY, max_attempts=MAX_POLLING_ATTEMPTS
             )
             logger.info("Pipeline completed successfully.")
+
+    def get_pipeline_run_metadata(
+        self, run_id: UUID
+    ) -> Dict[str, "MetadataType"]:
+        """Get general component-specific metadata for a pipeline run.
+
+        Args:
+            run_id: The ID of the pipeline run.
+
+        Returns:
+            A dictionary of metadata.
+        """
+        # TODO: Add this once we can get the region
+        # run_url = (
+        #     f"https://{region}.console.aws.amazon.com/sagemaker/"
+        #     f"home?region={region}"
+        # )
```

Comment:
Just a reminder in case this should make it into this PR

Reply:
This TODO will probably stay open till we have `zenml pipeline deploy` implemented